README.md

```diff
@@ -13,6 +11,20 @@ Additional information is available at the website: https://fwupd.org/
 
 The most up to date compilation instructions are available in the [Wiki](https://github.com/fwupd/fwupd/wiki/Compilation).
 
+**NOTE:** In most cases end users should never compile fwupd from scratch; it's a
+complicated project with dozens of dependencies (and as many configuration options)
+and there's just too many things that can go wrong.
+
+Users should just have fwupd installed and updated by their distro, hopefully
+by a packager who does understand all the config options.
+The distribution will have also done some testing with how fwupd interacts with
+other software on your system, for instance using GNOME Software.
+
+Installing fwupd from the [Snap Store](https://snapcraft.io/fwupd) or from
```

Comment:
I sorta think we should kick people here instead: https://github.com/fwupd/fwupd/wiki/fwupd-snap

Reply:
Agree, fixed.